gempy/core/model.py

```diff
@@ -1355,4 +1358,22 @@ def set_gravity_interpolator(self, density_block=None,
         if compile_theano is True:
             self.interpolator_gravity.compile_th_fn_grav(density_block, pos_density, inplace=True)
 
-        return self.additional_data.options
\ No newline at end of file
+        return self.additional_data.options
+
+    @property
+    def crs(self) -> str:
+        """Return coordinate reference system code of project
+
+        Returns: crs_code (str)
+
+        """
+        return self.__crs
```

Comment:
I think you can just use 'self.crs' instead of a private variable with getters and setters, as you are not really gaining anything with getters and setters.

Reply:
We want to be able to add checks (e.g. if a crs exists) and potentially additional functionality (e.g. adjusting views etc. when the coordinate system is changed).